kex-core/src/main/kotlin/org/vorpal/research/kex/parameters/Parameters.kt

```diff
@@ -92,3 +95,58 @@ fun Parameters<Descriptor>.filterIgnoredStatic(): Parameters<Descriptor> {
         }
     return Parameters(instance, arguments, filteredStatics)
 }
+
+fun Parameters<Descriptor>.replaceNullsWithDefaultValues(method: Method, cm: ClassManager): Parameters<Descriptor> {
+    val parametersWithoutNulls = arguments
+        .zip(method.argTypes)
+        .map { (descriptor, type) -> transformExistingDescriptor(descriptor, type.kexType, cm) }
+    return Parameters(instance, parametersWithoutNulls, statics)
+}
+
+private fun transformExistingDescriptor(descriptor: Descriptor, type: KexType, cm: ClassManager): Descriptor {
+    return when (descriptor) {
+        is ConstantDescriptor.Null -> type.newInstance(cm)
+        is ConstantDescriptor -> descriptor
+        is ArrayDescriptor -> ArrayDescriptor(descriptor.elementType, descriptor.length)
+            .apply {
+                descriptor.elements.forEach { (index, value) ->
```

Comment:
With this implementation array indices that were not included in `descriptor.elements` may still be null. Is this ok for you?